settings.gradle

```diff
@@ -26,16 +47,14 @@ rootProject.name = "ballerinax-${projectName}"
 include ":checkstyle"
 include ":${projectName}-native"
 include ":${projectName}-ballerina"
-include ":${projectName}-examples"
 
 project(':checkstyle').projectDir = file("build-config${File.separator}checkstyle")
 project(":${projectName}-native").projectDir = file('native')
 project(":${projectName}-ballerina").projectDir = file('ballerina')
-project(":${projectName}-examples").projectDir = file('examples')
 
 gradleEnterprise {
     buildScan {
         termsOfServiceUrl = 'https://gradle.com/terms-of-service'
         termsOfServiceAgree = 'yes'
     }
-}
+}
```

Comment:
missing EoF newline

Reply:
Addressed in [4ae71d8](https://github.com/ballerina-platform/module-ballerinax-azure-service-bus/pull/205/commits/4ae71d81c47f3fb1ef327b86f3c4ff4cf43c4e83)